documentation/test/index.md

```diff
@@ -1,16 +1,63 @@
 # The `test` function
-The test function is a single test in your validations. It is similar to unit tests `it` or `test` function. `test` accepts the name of the test, the error message and the actual test function. If you explicitly return true, or if your enforce function passed correctly, it is assumed that the test did not fail.
+The test function is a single test in your validations. It is similar to unit tests `it` or `test` function. `test` accepts the name of the test, the error message and the actual test logic. Tests can either be sync or async.
```

Comment:
"It is similar to unit tests `it` or `test` function." are you sur you want to expose this parallel?

Reply:
The whole premise of passable is that its syntax is similar to unit testing.